theMoth.py

```diff
@@ -19,15 +19,46 @@ async def on_message(message):
         await message.channel.send('Hello!')
 
     if message.content.startswith('?help'):
-        await message.channel.send('Supported commands:\n`?sooth` - draws a random sooth card\n`?roll` - rolls dice, e.g. `?roll 2d6` or `?roll d20 +5`\n\nWith no arguments, `?roll` will roll a mundane die from Invisible Sun, and with a +[number] argument it will roll a mundane die plus that many magic dice, e.g. `?roll +3`')
+        await message.channel.send('Supported commands:'

```

Comment:
Thanks for the formatting (here and on 123-126)! Didn't know you could do that.

Reply:
Learned it when making long helpstrings in hemppa.